tests/default_vars.sh

```diff
@@ -1187,7 +1188,7 @@ export_ugwpv1() {
   esac
 
   if [[ ${DO_GSL_DRAG_SS} = .true. ]]; then export CDMBGWD=${CDMBGWD_GSL}; fi
-  if [[ ${SEDI_SEMI} = .true. ]]; then export DT_ATMOS=$((DT_ATMOS/2)); fi
+  if [[ ${SEDI_SEMI} = .false. ]]; then export DT_ATMOS=$((DT_ATMOS/2)); fi
```

Comment:
I think DT_ATMOS stays the same and DT_INNER changes:
`if [[ ${SEDI_SEMI} = .false. ]]; then export DT_INNER=$((DT_ATMOS/2)); fi`

dt_inner in ccpp is for subcycling in the Thompson MP scheme

Reply:
You're right, got it fixed and it doesn't affect any of the RTs. Leaving the check in even if it doesn't get triggered for GW fidelity and reference.